content/docs/user-guide/glossary/dvc-cache.md

```diff
@@ -0,0 +1,22 @@
+---
+name: 'DVC Cache'
+match: ['DVC cache', 'cache', 'caches', 'cached']
+tooltip: >-
+  The DVC cache is a hidden storage (by default located in the `.dvc/cache`
+  directory) for files that are tracked by DVC, and their different versions.
+  Learn more about it's
+  [structure](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory).Learn
+  more about the [concept of cache](/doc/user-guide/glossary/dvc-cache) in DVC.
```

Comment:
> Learn more about the concept of cache

Prob no need to use the term "concept". These links could even be more casual e.g. link "DVC cache" at the beginning of the p (this way you also avoid repeating "Learn more about").

Reply:
OK, updated this and also tried something similar for workspace.

Reply:
Thanks.